js/views/file_input_view.js

```diff
@@ -103,11 +103,15 @@
                     var blob;
                     do {
                         i = i - 1;
-                        // dataURLtoBlob -> components/blueimp-canvas-to-blob
-                        blob = dataURLtoBlob(
+                        // TODO: Replace with native `Canvas::toBlob`:
+                        // https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob
+                        blob = window.dataURLToBlobSync(
```

Comment:
Renamed the original `dataURLtoBlob` (sic! weird `camelCase`) to `dataURLToBlobSync` since we now have an async `dataURLToBlob` from `blob-util` package.

Reply:
When would we moe to that native `Canvas::toBlob`? Let's really push for dealing with TODOs immediately. If we do leave a TODO in the code, we should have a good reason and we should create an issue to track that needed change.